### DISCUSSION TITLE
Can I use NFS-Ganesha to access existing CephFS volumes from a server ? (not a pod)

### DISCUSSION THREAD

**Discussion by freeeflyer** [rook/rook#15163] [Q&A]
Hi Everyone,

We are using a development K8S cluster with 
- rook-ceph
- metallb
- nginx-ingress 

Some developers would like to access CephFS volumes from their VM.
Can I use the NFS Ganesha integrated in Ceph to export them those volumes ?

Thanks

**Comment by travisn** [comment]
Yes, this should work. Try the topic on [accessing the nfs outside the K8s cluster](https://rook.io/docs/rook/latest-release/Storage-Configuration/NFS/nfs/#exposing-the-nfs-server-outside-of-the-kubernetes-cluster).
